### DISCUSSION TITLE
Masterchef for rebase tokens

### DISCUSSION THREAD

**Discussion by call3hudson** [call3hudson/masterchef-example#2] [Announcements]
For most LSTs including stETH for Lido, rebase tokens are widely used
Want to ensure if the masterchef is a good approach

**Comment by in-eth** [comment]
I think it's better to make document so that the devs and users can easily understand.

**Reply by call3hudson** [reply]
wdym?

**Reply by in-eth** [reply]
I'll update the doc.md file for that.

**Reply by in-eth** [reply]
Masterchef is a common method.

**Reply by dazhlord** [reply]
agree.
masterchef is common but very important concaept in blockchain world.
Many defi protocols use this basic model or upgraded one.
